File: check_schema.py
#!/usr/bin/env python3
import sys
from pathlib import Path

_project_root = Path(__file__).parent
if str(_project_root) not in sys.path:
    sys.path.insert(0, str(_project_root))

from utils.dotenv_loader import load_env_local
load_env_local()

from utils.db import DatabaseContext

# Get table schema
with DatabaseContext('read') as cur:
    cur.execute('''
        SELECT column_name, data_type, is_nullable
        FROM information_schema.columns
        WHERE table_name = 'orchestrator_execution_log'
        ORDER BY ordinal_position
    ''')

    rows = cur.fetchall()
    print(f'orchestrator_execution_log columns ({len(rows)} total):\n')
    for row in rows:
        col_name = row['column_name']
        col_type = row['data_type']
        nullable = row['is_nullable']
        print(f"  {col_name:30} {col_type:20} nullable={nullable}")

File: check-schema.js
const { query } = require('./webapp/lambda/utils/database');

(async () => {
  try {
    process.env.DB_HOST = 'localhost';
    process.env.DB_PORT = '5432';
    process.env.DB_USER = 'stocks';
    process.env.DB_PASSWORD = 'bed0elAn';
    process.env.DB_NAME = 'stocks';
    process.env.DB_SSL = 'false';

    const result = await query(`
      SELECT column_name, data_type 
      FROM information_schema.columns 
      WHERE table_name = 'price_daily'
      ORDER BY ordinal_position
    `);
    
    console.log('\nPrice Daily Schema:');
    result?.rows?.forEach(r => console.log(`  ${r.column_name.padEnd(25)} ${r.data_type}`));
    console.log();
    process.exit(0);
  } catch (err) {
    console.error('Error:', err.message);
    process.exit(1);
  }
})();
